src/platform/ModulesManager.c

```diff
@@ -317,7 +315,7 @@ static char* GenerateUuid(void)
     int random = 0;
     char c = ' ';
     int i = 0;
-    ssize_t size = UUID_LENGTH + 1;
+    ssize_t size = sizeof(uuidTemplate);
```

Comment:
>sizeof(uuidTemplate); [](http://example.com/codeflow?start=19&length=21)

Do ARRAY_SIZE instead. If this code will ever move to multi-byte characters, this will be broken.

Reply:
Also, better to rename 'size' to 'length' as it describes number of characters, not necessarily number of bytes (the two coincide here but that is not the rule)